tests/install_upgrade_operators/product_install/test_install_openshift_virtualization.py

```
@@ -170,6 +175,16 @@ def test_cnv_resources_installed_namespace_scoped(
         raise ResourceMismatch(f"Unexpected namespaced resources found post cnv installation: {mismatch_namespaced}")
 
 
+@pytest.mark.polarion("CNV-12453")
+@pytest.mark.order(after=CNV_INSTALLATION_TEST)
+# Dependency: CNV must be installed before storage class configuration can be verified
+@pytest.mark.dependency(depends=[CNV_INSTALLATION_TEST])
+def test_default_storage_class_set(admin_client, golden_images_namespace, updated_default_storage_class_from_config):
```

Ruff (0.14.13)

Unused function argument: `updated_default_storage_class_from_config`

(ARG001)